build/storage/tests/it/ptf_tests/system_tools/test_platform.py

```diff
@@ -96,62 +155,165 @@ def _install_docker(self):
     def check_system_setup(self):
         """Overwrite this method in specific platform if you don't want check all setup"""
         if not self._is_virtualization():
-            raise Exception("Virtualization is not setting properly")
+            raise DependencyException("Virtualization is not setting properly")
```

Comment:
You still return the same exception type. Exception messages are the same as the ones which you corrected in errors.py